tests/test_lib_events.py

```diff
@@ -2147,6 +2173,8 @@ def test_02_sendmail(self):
         un_handler = UserNotificationEventHandler()
         res = un_handler.do("sendmail", options=options)
         self.assertTrue(res)
+        msg = smtpmock.get_sent_message()
+        assert 'To: user@localhost.localdomain' in msg
```

Comment:
Please use 

    self.assertIn()

in tests.

Reply:
Done in bb386797